velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3642,12 +3642,19 @@ TEST_F(DateTimeFunctionsTest, timestampWithTimezoneComparisons) {
     test::assertEqualVectors(expectedResult, actual);
   };
 
+  /// Timestamp with timezone is internally represented with the milliseconds
+  /// already converted to UTC and thus normalized. The timezone does not play
+  /// a role in the comparison.
+  /// For example, 1970-01-01-06:00:00+02:00 is stored as
+  /// TIMESTAMP WITH TIMEZONE value (14400000, 960). 960 being the tzid
+  /// representing +02:00. And 1970-01-01-04:00:00+00:00 is stored as (14400000,
+  /// 0). These timestamps are equivalent.
   RowVectorPtr timestampWithTimezoneLhs = makeTimestampWithTimeZoneVector(
-      makeFlatVector<int64_t>({0, 0, 0}),
+      makeFlatVector<int64_t>({0, 0, 1000}),
       makeFlatVector<int16_t>({900, 900, 800}));
   RowVectorPtr timestampWithTimezoneRhs = makeTimestampWithTimeZoneVector(
       makeFlatVector<int64_t>({0, 1000, 0}),
-      makeFlatVector<int16_t>({900, 900, 900}));
```

Comment:
Do we need this change as the timezone values are unused?

Reply:
I did this to point out that the timezone has no bearing on the comparison. So I didn't want the first entry to have the exact same tzid which was required for the test to succeed.

Reply:
Can you add a comment to clarify this in the code as well?

Reply:
I see that you already added this on the top!